cli/ballerina-tool/src/main/resources/cli-help/ballerina-add.help

```diff
@@ -1,16 +1,17 @@
 NAME
-       Ballerina add - Add a Ballerina module in a project
+       ballerina add - Add a new Ballerina module to a project
```

Comment:
```suggestion
       ballerina add - add a new module to a Ballerina project
```

Reply:
Shouldn't it be a valid complete sentence?

Reply:
`man cat` produces something like

```
$ man cat
NAME
       cat - concatenate files and print on the standard output
```

If it starts with an upper case letter we have to finish with a full-stop too?